frontend/src/ArticleEdit/styled.js

```diff
@@ -87,3 +87,8 @@ export const Wrapper = styled.div`
   max-height: calc(100vh - 60px);
   max-width: 60rem;
 `;
+
+export const Message = styled.div`
+  visibility: ${props => (props.show ? 'visible' : 'hidden')};
+  color: ${props => (props.error ? 'tomato' : 'gold')};
+`;
```

Comment:
I prefer to use just `p` instead of `props`.
`gold` is not readable on white background.

Reply:
ok, don't mind changing it.
re gold: shade of green?

Reply:
Yeah, green-ish seems logical.